src/app/cube/about-us/components/timeline/events.json

```diff
@@ -0,0 +1,36 @@
+/* tslint:disable */
+[
+      {
+          "id": 1,
+          "date": "Date",
+          "description": "684 Learning Objects 174 Users NSA NCCP NSF C5",
```

Comment:
Can we add a small README in this module to explain how to add a new part of the timeline?